pkg/cmd/registry/artifacts/download/download.go

```diff
@@ -0,0 +1,170 @@
+package download
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+	"os"
+
+	flagutil "github.com/redhat-developer/app-services-cli/pkg/cmdutil/flags"
+	"github.com/redhat-developer/app-services-cli/pkg/connection"
+	"github.com/redhat-developer/app-services-cli/pkg/iostreams"
+	"github.com/redhat-developer/app-services-cli/pkg/localize"
+	"github.com/redhat-developer/app-services-cli/pkg/serviceregistry/registryinstanceerror"
+	"github.com/spf13/cobra"
+
+	"github.com/redhat-developer/app-services-cli/internal/config"
+	"github.com/redhat-developer/app-services-cli/pkg/cmd/factory"
+	"github.com/redhat-developer/app-services-cli/pkg/cmd/registry/artifacts/util"
+
+	"github.com/redhat-developer/app-services-cli/pkg/logging"
+)
+
+type Options struct {
+	group string
+
+	contentId  int64
+	globalId   int64
+	hash       string
+	outputFile string
+
+	registryID string
+
+	IO         *iostreams.IOStreams
+	Config     config.IConfig
+	Logger     func() (logging.Logger, error)
+	Connection factory.ConnectionFunc
+	localizer  localize.Localizer
+}
+
+func NewGetCommand(f *factory.Factory) *cobra.Command {
+	opts := &Options{
+		Config:     f.Config,
+		Connection: f.Connection,
+		IO:         f.IOStreams,
+		localizer:  f.Localizer,
+		Logger:     f.Logger,
+	}
+
+	cmd := &cobra.Command{
+		Use:   "get",
```

Comment:
This has the same command name as `get` so it is being overridden I guess?

Reply:
strangely enough I have changed it locally. Github outage prevented me to update it sorry.

Reply:
yes. IT was outdated. Github glitch